### PR TITLE
Native encrypt

### PR DESCRIPTION
**Pull request by mdhorn**
Changes proposed in this pull request:
- Switch to using go-native code for encryption

### CODE REVIEW

File: encrypt/encrypt.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+// SPDX-License-Identifier: GPL-3.0-only
+
+package encrypt
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	"github.com/GehirnInc/crypt"
+	// package requires import the hash method to blank
```
Comment (reaganlo) [inline encrypt/encrypt.go]: Typo in comment. "import" ==> "importing"

File: encrypt/encrypt.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	salt[2] = '$'
+
+	for i := 3; i < saltBytes; i++ {
+		salt[i] = dict[salt[i]%byte(len(dict))]
+	}
+
+	return string(salt), nil
+}
+
+// Crypt take a password and hashes with a random salt using SHA512
```
Comment (reaganlo) [inline encrypt/encrypt.go]: Typo in comment. "take" ==> "takes"

File: encrypt/encrypt_test.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package encrypt
+
+import (
+	"testing"
+
+	"github.com/GehirnInc/crypt"
+	// package requires import the hash method to blank
```
Comment (reaganlo) [inline encrypt/encrypt_test.go]: Typo in comment. "import" ==> "importing"

File: encrypt/encrypt.go (hunk 1 above)
Comment (mdhorn) [reply encrypt/encrypt.go]: done.

File: encrypt/encrypt_test.go (hunk 3 above)
Comment (mdhorn) [reply encrypt/encrypt_test.go]: done

File: encrypt/encrypt.go (hunk 2 above)
Comment (mdhorn) [reply encrypt/encrypt.go]: done.

**Pull request merged**